custom_components/easee/config_flow.py

```diff
@@ -129,4 +130,9 @@ async def async_step_init(self, user_input=None):
 
     async def _update_options(self):
         """Update config entry options."""
+        to_remove = []
```

Comment:
Could be written
```
self.hass.data[DOMAIN]["entities_to_remove"] = [cond for cond in self.prev_options[CONF_MONITORED_CONDITIONS] if conf not in self.options[CONF_MONITORED_CONDITIONS]] 
```

Reply:
OK, I will change. I'm not speaking fluent pythonic yet. Should be` if cond not in ` ??